BassBotDotNET/Program.cs

```diff
@@ -87,13 +84,21 @@ public async Task Client_Ready()
         globalCommand.WithName("first-global-command");
         globalCommand.WithDescription("This is my first global slash command");
 
+        List<ApplicationCommandProperties> applicationCommandProperties = new();
+
         try
         {
-            // Now that we have our builder, we can call the CreateApplicationCommandAsync method to make our slash command.
-            await guild.CreateApplicationCommandAsync(firstGuildCommand.Build());
-            
-            await _client.Rest.CreateGuildCommand(listRolesGuild.Build(), guildId);
-            await _client.Rest.CreateGuildCommand(_settingSlashCommandModule.Command().Build(), guildId);
+            // We can group our commands together with application command properties.
+            SlashCommandBuilder feedbackCommand = _feedbackSlashCommandModule.Command();
+            SlashCommandBuilder slashCommandBuilder = _settingSlashCommandModule.Command();
+            SlashCommandBuilder listRolesCommand = listRolesGuild;
+            SlashCommandBuilder firstGuildcommandBuilder = firstGuildCommand;
+
+            applicationCommandProperties.Add(feedbackCommand.Build());
+            applicationCommandProperties.Add(slashCommandBuilder.Build());
+            applicationCommandProperties.Add(listRolesCommand.Build());
+            applicationCommandProperties.Add(firstGuildcommandBuilder.Build());
+
```

Comment:
one thing im a bit concerned about is that this may be for global commands only and guild commands cannot do this................................... The doc said this is for bulk GLOBAL commands but im not sure if theres a guild command version.

Reply:
Just tested. the commands differentiate between guild and global based on how the _client or guild adds the command. And unfortunately, bulk adding only works on global.